docs/index.html

```diff
@@ -272,15 +266,25 @@
           EditOnGithubPlugin.create(
             'https://github.com/oeway/ImJoy/blob/master/docs/'
           )
-        ]
+        ],
+        tabs: {
+            persist    : true,      // default
+            sync       : true,      // default
+            theme      : 'classic', // default
+            tabComments: true,      // default
+            tabHeadings: true       // default
+          }
       };
     </script>
     <script src="/docs/static/docsify.min.js"></script>
     <script src="/docs/static/prism-bash.min.js"></script>
     <script src="/docs/static/prism-json.min.js"></script>
+    <script src="/docs/static/prism-python.min.js"></script>
     <script src="/docs/static/search.min.js"></script>
     <script src="/docs/static/external-script.min.js"></script>
     <script src="/docs/static/docsify-copy-code.min.js"></script>
     <script src="/docs/static/docsify-pagination.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/docsify-tabs@1"></script>
```

Comment:
For supporting offline access, these two files should be downloaded to /docs/static and replace with a local url.